Data Structures/Array/dynamic_array.cpp

```diff
@@ -0,0 +1,86 @@
+//Name: Kumar Raju.B
+//E-mail: bim2017008@iiita.ac.in
+//TITLE: DYNAMIC ARRAY-VECTOR(STL)
+#include<bits/stdc++.h>
+using namespace std;
+void display_array( vector<int> v)
```

Comment:
Follow [Coding convention for function name](https://github.com/sahilbansal17/Competitive_Coding/wiki/Coding-Standards#naming-convention)